src/otp-input/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ButtonProps } from "src/button";
+
+export interface OtpInputProps extends React.AriaAttributes {
+    value?: string | undefined;
+    error?: boolean | undefined;
+    errorMessage?: string | undefined;
+    "data-testid"?: string | undefined;
+    cooldownDuration: number | undefined;
+    ctaProps: ButtonProps | undefined;
+    numOfInput: number | undefined;
+    onChange?: ((value: string) => void) | undefined;
+
+    // Input props
```

Comment:
None of the props here seem to be relevant. I think they can be omitted

Reply:
I think might need some of later in form. but will remove for now.